int64 jointAngles
---
int64 mimicAngles